domain/src/main/java/org/oppia/android/domain/topic/StoryProgressTestHelper.kt

```diff
@@ -100,6 +99,88 @@ class StoryProgressTestHelper @Inject constructor(
     )
   }
 
+  /** Marks full story progress for a particular profile. */
+  fun markFullProgressForAllTopics(profileId: ProfileId, timestampOlderThanAWeek: Boolean) {
+    val timestamp = if (!timestampOlderThanAWeek) {
+      getCurrentTimestamp()
+    } else {
+      getOldTimestamp()
+    }
+    markFullTopicProgressForFractions(profileId, timestampOlderThanAWeek)
```

Comment:
I suggest organizing groups of the following recordCompletedChapter blocks into subsequent topic-based methods like fractions (e.g. one for test topic, ratios, etc.) for consistency and cleanliness.

Reply:
Done